auth/src/express/middleware/parser.ts

```diff
@@ -0,0 +1,25 @@
+import { NextFunction, Request, Response } from 'express';
+
+import { Logger } from '../../api/services/logger';
+
+export namespace Parser {
+    function isValidJsonFormat(input: any): boolean {
+        return typeof input === 'string' && (input.trim().startsWith('{') || input.trim().startsWith('['));
```

Comment:
I would propose to accept `null`, `undefined`, `""` and a string srtating with `{`. The three first cases should be converted (can also be done by the parser, see another comment) to `{}`, and the forth one can be parsed. If there was an error, a `ParserError` is thrown.

Reply:
Also I would expect the middleware to be a clean invariant: Iff the request payload is ok, it is always a parsed object. If not, the middleware will provide a valid 400er response. With this invariant (object or nothing) the behaviour is very clear. I think there is no need to accept arrays, too.